### PR TITLE
build-scripts: oe: fix tools version

### PR DESCRIPTION
**Pull request by jean-edouard**
Signed-off-by: Jed <lejosnej@ainfosec.com>

### CODE REVIEW

File: build-scripts/oe/build.sh [hunk 1]
```diff
@@ -65,7 +65,7 @@ XENCLIENT_BUILD_DATE = "`date +'%T %D'`"
 XENCLIENT_BUILD_BRANCH = "${BRANCH}"
 XENCLIENT_VERSION = "$VERSION"
 XENCLIENT_RELEASE = "$RELEASE"
-XENCLIENT_TOOLS = "$XENCLIENT_TOOLS"
```
Comment (rossphilipson) [inline build-scripts/oe/build.sh]: What is the value of XENCLIENT_TOOLS and where does it come from?

Comment (jean-edouard) [inline build-scripts/oe/build.sh]: It doesn't exist, that's why this line is wrong...

Comment (rossphilipson) [inline build-scripts/oe/build.sh]: Lol. So what was the symptom - the Windows tools were not getting a version number?

Comment (jean-edouard) [inline build-scripts/oe/build.sh]: The WIndows tools will have a version but it won't be in /etc/xenclient.conf.
Surprisingly enough, the "no tools" icon still disappears after installing the tools though, so no real user symptom.

Comment (rossphilipson) [inline build-scripts/oe/build.sh]: Ack, merging soon.

**Pull request merged**